app/dashboard/templates/dashboard/hackathon/new_bounty.html

```diff
@@ -513,10 +519,6 @@ <h5 class="text-uppercase h3 font-weight-bold mb-0">Total</h5>
     <script src="{% static "v2/js/ipfs.js" %}"></script>
     <script src="{% static "v2/js/abi.js" %}"></script>
 
-    <script src="{% static "v2/js/lib/polkadot/core.min.js" %}"></script>
-    <script src="{% static "v2/js/lib/polkadot/extension.min.js" %}"></script>
-    <script src="{% static "v2/js/lib/polkadot/utils.js" %}"></script>
```

Comment:
Why are we removing these?

Reply:
same as https://github.com/gitcoinco/web/pull/8259#discussion_r575626847